### PR TITLE
feat: added dim config for rolling_cohorts view

### PR DESCRIPTION
**Pull request by kik-kik**
# feat: added dim config for rolling_cohorts view

### CODE REVIEW

**Review by gkabbz — APPROVED** [review]

File: dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
```
Comment (lucia-vargas-a) [inline dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml:3]: Kik, what does it mean that I am the owner / who will receive notifications?

File: dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
```
Comment (lucia-vargas-a) [inline dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml:14]: Same comment as in https://github.com/mozilla/dim/pull/42/files#r1028095099

File: dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    - type: not_null
+      params:
+        columns:
+          - cohort_date
+          - client_id
```
Comment (lucia-vargas-a) [inline dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml:19]: Do you also want to check here when the length of client_id is different than expected?

File: dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+          - infrequent_user
+          - other
+          - casual_user
+          - regular_user
+          - core_user
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_app_name
+        expected_values:
```
Comment (lucia-vargas-a) [inline dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml:60]: Some values missing? Here is the full list [from PR34](https://github.com/mozilla/dim/blob/2fd107e4349d8fc70ce0538a905cd2ac9430a188/dim_checks/moz-fx-data-shared-prod/telemetry/unified_metrics/dim_checks.yaml#L17)

File: dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+          - Linux_CXL
+          - Darwin
+          - Windows_95
+          - Windows_98
+          - Windows_NT
+          - OpenBSD
+          - SunOS
+          - HubertOS
+
+    # TODO: Regex check to make sure locale follows: [2 characters]-[2 characters] rule
```
Comment (lucia-vargas-a) [inline dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml:85]: Is [this the validation](https://github.com/mozilla/dim/blob/54bc34e71e5af3824429639767adcde9f87f5745/dim_checks/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/dim_checks.yaml#L84) you mean to add here?

**Review by lucia-vargas-a — CHANGES_REQUESTED** [review]

File: dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml (hunk 1 above)
Comment (kik-kik) [reply dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml:3]: Answered in the other PR.

Comment (kik-kik) [reply dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml:3]: https://github.com/mozilla/dim/pull/42/files#r1028239357

**Pull request merged**